gateway/1.4/modules/ROOT/pages/flex-gateway-publish-apis-conn.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-For Flex Gateway running in Connected Mode, you publish APIs using *API Manager*. Refer to xref:api-manager::create-instance-task.adoc[Adding an API Instance] for more information.
+For Flex Gateway running in Connected Mode, you publish APIs using *API Manager*. Refer to xref:api-manager::create-instance-task-flex.adoc[] for more information.
```

Comment:
Does this resolve correctly when you build? I wasn't able to get cross-repo cross-references to resolve using the empty brackets. Is there a trick to it?